webapp/src/Controller/API/SubmissionController.php

```diff
@@ -236,6 +236,7 @@ public function addSubmissionAction(Request $request, ?string $id): Response
 
         // By default, use the team of the user
         $team = $this->dj->getUser()->getTeam();
+        $user = $this->dj->getUser();
```

Comment:
nit: switch this with the previous line and then write it as `$team = $user->getTeam();`